lib/inferno/db/migrations/010_add_validator_sessions.rb

```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
+  change do
+    create_table :validator_sessions do
+      column :id, String, primary_key: true, null: false, size: 36
+      column :validator_session_id,  String, null: false, size: 255, unique: true
+      column :test_suite_id, String, null: false, size: 255
+      column :validator_name, String, null: false, size: 255
+      column :suite_options, String, text: true, size: 255
+      column :last_accessed, DateTime, null: false
+      index [:validator_name, :test_suite_id, :suite_options]
```

Comment:
I believe the columns in the index have to be in the same order as they are in the query in order for the index to work.